e2e/volume_ops_test.go

```diff
@@ -271,6 +272,39 @@ func testVolumeDelete(t *testing.T) {
 func testVolumeStart(t *testing.T) {
 	r := require.New(t)
 	r.Nil(client.VolumeStart(volname, false), "volume start failed")
+
+	vol, err := client.BricksStatus(volname)
+	for index := range vol {
+		r.NotZero(vol[index].Port)
+		r.True(vol[index].Online)
+	}
+
+	vol, err = client.BricksStatus(volname)
+	killBrick := vol[0].Info.Path
+	process, err := os.FindProcess(vol[0].Pid)
+	r.Nil(err, fmt.Sprintf("failed to find brick pid: %s", err))
```

Comment:
`r.Nil()` will print error anyway. It'll be more useful if your error string prints pid and path.

Reply:
Done